pkg/actors/actors.go

```diff
@@ -377,8 +377,8 @@ func (a *actorsRuntime) callRemoteActor(
 	ctx context.Context,
 	targetAddress, targetID string,
 	req *invokev1.InvokeMethodRequest) (*invokev1.InvokeMethodResponse, error) {
-	log.Infof("Entering callRemoteActor: %s - %s", req.Actor().GetActorType(), req.Actor().GetActorId())
-	log.Infof("callRemoteActor Request Headers: %v", req.Metadata())
+	log.Debugf("Entering callRemoteActor: %s - %s", req.Actor().GetActorType(), req.Actor().GetActorId())
+	log.Debugf("callRemoteActor Request Headers: %v", req.Metadata())
```

Comment:
even in debug do we need to print the request headers?

Reply:
They become important in actor reentrancy, but not really anywhere else I think. I figured if it was there, why not?

I am also fine with removing both of these lines and adding in more debugging logs as an actual issue for 1.3.

Reply:
One of the possible why nots is leaking headers related to authentication in the logs.

Reply:
Valid, which is why I'm leaning more towards just removing these and taking a more thought out action for 1.3 to give some general debugging for actors.

What do you think @mukundansundar @wcs1only?

Reply:
Removed the logging.